pkg/kubelet/network/cni/cni.go

```diff
@@ -319,6 +263,10 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 	glog.V(4).Infof("Got netns path %v", podNetnsPath)
 	glog.V(4).Infof("Using netns path %v", podNs)
 
+	if podNetnsPath == "" {
+		podNetnsPath = "none"
```

Comment:
"none" is theoretically a valid netns path on Linux - can you just use the empty string?